packages/lesmis/src/api/queries/useOrgUnitData.js

```diff
@@ -35,3 +35,6 @@ export const useCurrentOrgUnitData = ({ includeCountryData = false } = {}) => {
   const { organisationUnitCode } = useUrlParams();
   return useOrgUnitData({ organisationUnitCode, includeCountryData });
 };
+
+export const useCountryHeirarchyData = () =>
```

Comment:
This will need to be updated soon once the entity server descendants endpoint is ready.